games/minimal/mods/testentities/mod.conf

```diff
@@ -0,0 +1,2 @@
+name = testentities
```

Comment:
You could leave the `test` in the name away, just call it `entities` - this entire game is a testbed ;)